Day2/que2.js

```diff
@@ -0,0 +1,8 @@
+//  Write a function to take input(age, name, phone no.) from user and display it
+
+
+let age = prompt('Enter your age');
+let name = prompt('Enter your namee');
+let phoneNo = prompt('Enter your phone number');
+
+console.log(`  Entered name is ${name} Phone no is ${phoneNo} and mail is ${email}`);
```

Comment:
email is not defined. This should have given an error. Seems the code isn't tested. Please test all code for its functionality, so that such bugs aren't there

Reply:
@anantmishra-successive

Reply:
Please do testing for all questions, all PRs.

Reply:
![output](https://github.com/anantmishra-successive/JS_Training/assets/162098832/7beeb10b-1d0d-4839-9aae-0986da194592)
Done Testing